### DISCUSSION TITLE
How to place button in right position.

### DISCUSSION THREAD

**Discussion by Rishav1707** [open-sauced-craftwork/browser-extension#19] [Q&A] [answered]
## I created the  button but am unable to place it in its proper location. It overlaps with the biotext. Could someone please tell me how to do that?

https://user-images.githubusercontent.com/97666287/224963438-1f89a237-e376-4484-85c7-1004880400bc.mp4

**Comment by bdougie** [comment]
This will take trial and error. Consider what id you are using to query the DOM. It seems like what every you are selecting in not ideal and not respecting other elements around it.

**Reply by Rishav1707** [reply]
I didn't understand what you are trying to say

**Reply by bdougie** [reply]
Nick provided a better answer below.

**Comment by nickytonline** [comment, marked as answer]
The button has an identifier. It can be accessed via a [CSS Selector](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors) when [querying the document object model](https://developer.mozilla.org/en-US/docs/web/api/document/queryselector) (DOM) as @bdougie mentions. For positioning, check out the documentation on MDN for the CSS [position property.](https://developer.mozilla.org/en-US/docs/Web/CSS/position?v=example)

**Reply by Rishav1707** [reply]
But my question is how to place the button between the biotext and the EditProfile button?

**Reply by bdougie** [reply]
Your question is the answer to the assignment. Please leverage the resources provide to discover the answer.

**Reply by Rishav1707** [reply]
Ok thank you very much @bdougie
